test/integration/order_integration_test.rb

```diff
@@ -1,26 +1,45 @@
 require 'test_helper'
 
-class UserLoginTest < ActionDispatch::IntegrationTest
+class OrderIntegrationTest < ActionDispatch::IntegrationTest
   include Capybara::DSL
-  attr_reader :user
+  attr_reader :user, :order
 
   def setup
     @user = User.create(username: 'user',
-                      password: 'password',
-                      first_name: 'John',
-                      last_name: 'Doe',
-                      email: 'example@example.com')
+                        password: 'password',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.